samples/snippets/src/test/java/com/example/bigtable/deletes/DeletesTest.java

```diff
@@ -0,0 +1,165 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigtable.deletes;
+
+import com.example.bigtable.MobileTimeSeriesBaseTest;
+import com.google.api.gax.rpc.ServerStream;
+import com.google.cloud.bigtable.admin.v2.BigtableTableAdminClient;
+import com.google.cloud.bigtable.data.v2.BigtableDataClient;
+import com.google.cloud.bigtable.data.v2.models.Query;
+import com.google.cloud.bigtable.data.v2.models.Row;
+import com.google.cloud.bigtable.data.v2.models.RowCell;
+import com.google.common.truth.Truth;
+import java.io.IOException;
+import java.util.List;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+/*
+ * These tests are order dependent because they delete rows and cells from a table.
+ * They are prefixed with `testN_` to signal the order in which they should run.
+ */
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class DeletesTest extends MobileTimeSeriesBaseTest {
+  public static BigtableDataClient bigtableDataClient;
+
+  @BeforeClass
+  public static void beforeClass() throws IOException {
+    initializeVariables();
+    createTable();
+    writeStatsData();
+    writePlanData();
+    bigtableDataClient = BigtableDataClient.create(projectId, instanceId);
+  }
+
+  @AfterClass
+  public static void afterClass() throws IOException {
+    cleanupTable();
+  }
+
+  @Test
+  public void test1_testDeleteFromColumn() {
+    String rowKey = "phone#4c410523#20190501";
+    Row row = bigtableDataClient.readRow(TABLE_ID, rowKey);
+    String familyName = "cell_plan";
+    String qualifier = "data_plan_01gb";
+    List<RowCell> cells = row.getCells(familyName, qualifier);
+    Truth.assertThat(cells).isNotEmpty();
+    DeleteFromColumnExample deleteFromColumnExample = new DeleteFromColumnExample();
+    deleteFromColumnExample.deleteFromColumn(
+        projectId, instanceId, TABLE_ID, rowKey, familyName, qualifier);
+
+    row = bigtableDataClient.readRow(TABLE_ID, rowKey);
+    List<RowCell> cellsAfterDelete = row.getCells(familyName, qualifier);
+    Truth.assertThat(cellsAfterDelete).isEmpty();
+  }
+
+  @Test
+  public void test2_testDeleteFromRow() {
+    String rowKey = "phone#4c410523#20190501";
+    Row row = bigtableDataClient.readRow(TABLE_ID, rowKey);
+    Truth.assertThat(row).isNotNull();
+    DeleteFromRowExample deleteFromRowExample = new DeleteFromRowExample();
+    deleteFromRowExample.deleteFromRow(projectId, instanceId, TABLE_ID, rowKey);
+    row = bigtableDataClient.readRow(TABLE_ID, rowKey);
+    Truth.assertThat(row).isNull();
+  }
+
+  @Test
+  public void test3_testStreamingAndBatching() {
+    String rowKey = "phone#4c410523#20190502";
+    Row row = bigtableDataClient.readRow(TABLE_ID, rowKey);
+
+    String familyName = "cell_plan";
```

Comment:
perhaps making these a const at the top could be easier. Also they might be accessible from the base test? I don't remember

Reply:
replaced some variables with the static ones in the base test. Some, like 'rowKey' are only used in two places at once, so I left them as is.

Reply:
Looks good!